x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/logic/use_exception_flyout_data.tsx

```diff
@@ -97,6 +97,24 @@ export const useFetchIndexPatterns = (rules: Rule[] | null): ReturnUseFetchExcep
       if (activeSpaceId !== '' && memoDataViewId) {
         setDataViewLoading(true);
         const dv = await data.dataViews.get(memoDataViewId);
+
+        // the autosuggest api fails if the .alerts index does not exist.
+        // So we need to append '*' char at end of .alerts-security.alerts-{spaceId}
+        // to prevent this from happening.
+        const newPattern = dv
+          .getIndexPattern()
+          .split(',')
+          .reduce((acc: string[], item: string) => {
+            if (item.includes('.alerts')) {
+              acc.push(item.concat(`*`));
+            } else {
+              acc.push(item);
+            }
+            return acc;
+          }, [] as string[])
+          .join(',');
+        dv.setIndexPattern(newPattern);
```

Comment:
Does this then update the index pattern across the app, not just for this one call?

Reply:
Yeah I thought it was temporary but that is not the case. I'm pushing a commit that will create an ad-hoc data view based on the security solution default data view + the additional `*` char.